gserver/src/steps.handler.ts

```diff
@@ -359,7 +358,12 @@ export default class StepsHandler {
             //We can replace some outputs, ex. strings in brackets to make insert strings more neat
             res = res.replace(/\"\[\^\"\]\+\"/g, '""');
         }
-
+        
+        const favQuote = this.settings.cucumberautocomplete.favoriteQuoteChar;
+        if (favQuote){
```

Comment:
If you don't provide the favoriteQuoteChar setting, is it set to `"\""` by default, or is it set to null?

Isn't this functionality the type that the entire user base could benefit from? Rather than users having to update their cucumberautocomplete settings before they can benefit from it?

Reply:
it is set to null.
and produces usual output : ("|')${1:}\1

Reply:
oh sorry by default favoriteQuoteChar is set to "\""
I think it's time to suppress this option so

Reply:
If we remove this setting then this if statement is redundant, yes!